### PR TITLE
Add Asuswrt component and sensor

### PR DESCRIPTION
**Pull request by kennedyshead**
**Description:**
This is the documentation for the new Asuswrt component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17757

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/asuswrt.markdown [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+title: "Asuswrt"
+description: "Instructions on how to integrate Asuswrt into Home Assistant."
+date: 2017-01-05 17:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: asus.png
+ha_category: Hub
+featured: false
```
Comment (MartinHjelmare) [inline source/_components/asuswrt.markdown]: This is the default.

File: source/_components/asuswrt.markdown [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+ha_release: 0.82
```
Comment (MartinHjelmare) [inline source/_components/asuswrt.markdown]: 0.83

File: source/_components/asuswrt.markdown [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  username: YOUR_ADMIN_USERNAME
+```
+
+{% configuration %}
+host:
+  description: "The IP address of your router, eg. `192.168.1.1`."
+  required: true
+  type: string
+username:
+  description: "The username of an user with administrative privileges, usually `admin`."
```
Comment (MartinHjelmare) [inline source/_components/asuswrt.markdown]:
```suggestion
  description: "The username of a user with administrative privileges, usually `admin`."
```

File: source/_components/sensor.asuswrt.markdown [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+description: "Instructions on how to integrate Asuswrt sensors into Home Assistant."
+date: 2017-08-30 12:21
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: asus.png
+ha_category: Environment
+ha_iot_class: "Local Polling"
+ha_release: 0.82
```
Comment (MartinHjelmare) [inline source/_components/sensor.asuswrt.markdown]: 0.83

File: source/_components/sensor.asuswrt.markdown [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+The `melissa` platform allows you to get data from your [ASUSWRT](http://event.asus.com/2013/nw/ASUSWRT/) sensors from within Home Assistant.
```
Comment (MartinHjelmare) [inline source/_components/sensor.asuswrt.markdown]: This is not the melissa platform. :wink:

File: source/_components/sensor.melissa.markdown [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-layout: page
+layout: **page**
```
Comment (MartinHjelmare) [inline source/_components/sensor.melissa.markdown]: Probably revert this.

Comment (kennedyshead) [reply source/_components/sensor.melissa.markdown]: hmm i did not do this ://

**Review by MartinHjelmare — APPROVED** [review]
Looks good!

File: source/_components/sensor.asuswrt.markdown [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+layout: page
+title: "Asuswrt Sensor"
+description: "Instructions on how to integrate Asuswrt sensors into Home Assistant."
+date: 2017-08-30 12:21
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: asus.png
+ha_category: Environment
```
Comment (MartinHjelmare) [inline source/_components/sensor.asuswrt.markdown]: Is this correct or is it a leftover from the Melissa page?

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]
Fix the category of the sensor and rename the title in the front matter of the device tracker page similar as the sensor page.

File: source/_components/device_tracker.asuswrt.markdown [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
 title: "ASUSWRT"
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.asuswrt.markdown]: Change the title to eg "Asuswrt Device Tracker".

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**